index.md

```diff
@@ -32,25 +32,25 @@ The project strives to create an application interface which lowers the barrier
 to use the data for visualization and/or analysis.
 
 
-Tackling the Problem
---------------------
+Our Approach
+------------
 
 The project's core is an API which can be used to query the processed data.
 For this purpose we develop a cloud service which queries the freely
 accessible data of Sentinel-5P, aggregates it, stores it in a cache and makes
 it available.
 
 
-Target Audience
----------------
+Who is it for?
+--------------
 
 This project targets developers who want to build their own services based on
 the satellite data of the Copernicus program, but who do not want to work with
 huge amounts of scientific data directly. We will provide examples and
 libraries to quickly get you started without having to become an expert in
 satellite data analysis yourself.
 
-Publications
-------------
+In the News
```

Comment:
```suggestion
Media
```

Reply:
I actually like `In the News` more. It is hard to discuss about, since `Media` does describe the content of this part very well. I just have the feeling that `In the News` does sound fresher and more playful, but is still professional enough in my opinion. It sparks more joy :smile:

I hope this does make at least a little bit of sense.

Reply:
But the only thing that one could remotely call “the news” in this category is the FLOSS Weekly episode. Everything else is no news story but talks at conferences.

Reply:
`Us talking about emissions API`

I want things that spark joy. I like joy.

Other options:

`Us talking`, `Us talking dirty (because emissions, you know ;)`,  `Oldies but Goldies`, `Say my name, say my name` (although I cut that part from the video), `because we couldn't afford to be on TED`, `look, our english is funny`

Reply:
@tibroc also likes `Media` more. So I changed it, since joy seems not the important anymore.